optimum/intel/neural_compressor/modeling_base.py

```diff
@@ -111,8 +123,113 @@ def _from_pretrained(
         local_files_only: bool = False,
         subfolder: str = "",
         trust_remote_code: bool = False,
+        *model_args,
         **kwargs,
     ):
+        device_map = kwargs.get("device_map", "cpu")
+        use_xpu = True if device_map == torch.device("xpu") or device_map == "xpu" else False
+
+        config = kwargs.pop("config", None)
+
+        quantization_config = kwargs.pop("quantization_config", None)
+        if not isinstance(config, PretrainedConfig):
+            config, _ = AutoConfig.from_pretrained(
+                model_id,
+                return_unused_kwargs=True,
+                **kwargs,
+            )
+        if hasattr(config, "quantization_config"):
+            if config.quantization_config is None:
+                logger.warning(
+                    "Quantization_config loading failed. If you want to load saved "
+                    "low bit model, please check your quantizate_config.json."
+                )
+            else:
+                logger.info("quantization_config: {}".format(config.quantization_config))
+                try:
+                    from neural_compressor.transformers.models.modeling_auto import (
+                        _BaseQBitsAutoModelClass,
+                    )
+
+                    _BaseQBitsAutoModelClass.ORIG_MODEL = cls.auto_model_class
+                    model = _BaseQBitsAutoModelClass.load_low_bit(
+                        model_id,
+                        *model_args,
```

Comment:
shoudn't it be instead :
```suggestion
                        subfolder=subfolder,
                        revision=revision,
                        cache_dir=cache_dir,
                        token=token,
                        local_files_only=local_files_only,
                        force_download=force_download,
                        trust_remote_code=trust_remote_code,
```